website/content/docs/job-specification/consul.mdx

```diff
@@ -99,9 +104,7 @@ The [`template`][template] block can use the Consul token as well.
 - `partition` `(string: "")` - When this field is set, a constraint will be
   added to the group or task to ensure that the allocation is placed on a Nomad
   client that has a Consul Enterprise agent in the specified Consul [admin
-  partition][]. Note that Consul Community Edition agents are not assigned to
-  any admin partition, so this field should not be used without Consul
-  Enterprise.
+  partition][]. Note that Consul Community Edition agents are limited to the `default` partition, so only Consul Enterprise users should configure this field.
```

Comment:
Our testing while working on this feature showed that Consul CE doesn't return a fingerprint for the partition at all, not that it's in a "default" partition. The documentation we have here is correct for the way Nomad's fingerprinting works.

(Also, we hard-wrap markdown)